From cab0a8fb8d9a1095de3e4c2227bfc37fae93f781 Mon Sep 17 00:00:00 2001
From: Rich Felker <dalias@aerifal.cx>
Date: Sun, 30 Aug 2020 16:49:48 -0400
Subject: clean up overinclusion in files using TIOCGWINSZ

now that struct winsize is available via sys/ioctl.h once again,
including termios.h is not needed.
---
 src/stdio/__fdopen.c       | 1 -
 src/stdio/__stdout_write.c | 1 -
 src/unistd/isatty.c        | 1 -
 3 files changed, 3 deletions(-)

diff --git a/src/stdio/__fdopen.c b/src/stdio/__fdopen.c
index 616f4f99..116e78e5 100644
--- a/src/stdio/__fdopen.c
+++ b/src/stdio/__fdopen.c
@@ -1,7 +1,6 @@
 #include "stdio_impl.h"
 #include <stdlib.h>
 #include <sys/ioctl.h>
-#include <termios.h>
 #include <fcntl.h>
 #include <errno.h>
 #include <string.h>
diff --git a/src/stdio/__stdout_write.c b/src/stdio/__stdout_write.c
index 5b413c79..dd1ec60f 100644
--- a/src/stdio/__stdout_write.c
+++ b/src/stdio/__stdout_write.c
@@ -1,6 +1,5 @@
 #include "stdio_impl.h"
 #include <sys/ioctl.h>
-#include <termios.h>
 
 size_t __stdout_write(FILE *f, const unsigned char *buf, size_t len)
 {
diff --git a/src/unistd/isatty.c b/src/unistd/isatty.c
index bc220c00..75a9c186 100644
--- a/src/unistd/isatty.c
+++ b/src/unistd/isatty.c
@@ -1,7 +1,6 @@
 #include <unistd.h>
 #include <errno.h>
 #include <sys/ioctl.h>
-#include <termios.h>
 #include "syscall.h"
 
 int isatty(int fd)
-- 
cgit v1.2.1

